common/utils.go

```diff
@@ -152,3 +152,11 @@ func GetCiliumVersionString(epCHeaderFilePath string) (string, error) {
 		}
 	}
 }
+
+// RequireRootPrivilege checks if the user running cmd is root. If not, it exits the program
+func RequireRootPrivilege(cmd string) {
+	if os.Getuid() != 0 {
+		fmt.Fprintf(os.Stderr, "Please run %q commands with root privileges.\n", cmd)
```

Comment:
That would duplicate the exiting code across all of the callees of this function . All of the times I invoked this function, I want to exit, so I put the exit call in the function itself to avoid the duplication.